app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -212,6 +213,32 @@ class QuestionPlayerActivityTest {
     }
   }
 
+  @RunOn(TestPlatform.ROBOLECTRIC)
+  @Test
+  fun testQuestionPlayer_submitCorrectAnswer_correctTextBannerIsDisplayed() {
```

Comment:
What are these tests verifying that the more targeted fragment tests don't cover?

Reply:
These are the exact same, I wasn't sure if we wanted them in both files, but will remove these ones in preference of the LocalTests

Reply:
Done